.asf.yaml

```
@@ -35,6 +35,11 @@ github:
     dev:
       required_status_checks:
         strict: true
+        context:
+          - check-license
+          - CheckDeadLinks
+          - Build
+          - Spark
```

You should also add `Backend` to context, otherwise our backend build is not required
```suggestion 
          - Spark
          - Backend
```